unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Scripts/Sections/PlacesAndEventsSection/SubSections/EventsSubSection/EventsSubSectionMenu/EventsSubSectionComponentController.cs

```diff
@@ -382,7 +289,5 @@ internal void UnsubscribeToEvent(string eventId)
         //    {
         //        Debug.LogError($"Error posting 'attend' message to the API: {error}");
         //    });
```

Comment:
remove commented code

Reply:
For the moment I prefer to leave it like this (it has a `TODO` mark) in order to have the code reference when we implement the "Unsubscribe" API call in the future.